app-editors/nvi/nvi-1.81.6-r6.ebuild

```diff
@@ -43,22 +43,22 @@ RDEPEND="${CDEPEND}
 REQUIRED_USE="tcl? ( !unicode )"
 
 src_prepare() {
-
-	epatch "${FILESDIR}"/${P}-strlen-macro-renaming.patch
-	epatch "${FILESDIR}"/${P}-db44.patch
-	epatch "${FILESDIR}"/${P}-db.patch
-	epatch "${FILESDIR}"/${P}-perl-as-needed.patch
-	epatch "${FILESDIR}"/${P}-perl-shortnames.patch
-	epatch "${FILESDIR}"/${P}-ac_config_header.patch
-	epatch "${FILESDIR}"/${P}-use_pkgconfig_for_ncurses.patch
+	eapply "${FILESDIR}"/${P}-strlen-macro-renaming.patch
```

Comment:
don't use bajillions of `eapply`'s, just drop them into a bash array called `PATCHES` and then call default instead of `eapply` + `eapply_user`